WordPress/Classes/Utility/LocalizableStringsUtility.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  LocalizableStringsUtility.swift
+//  WordPress
+//
+//  Created by Evangelos Sismanidis on 29.09.17.
+//  Copyright © 2017 WordPress. All rights reserved.
+//
+
+import Foundation
+
+enum Strings: String {
```

Comment:
Perhaps a more meaningful name? `LocalizedStrings` _could_ work out, since the goal of this one appears to be... returning localized strings?

Reply:
+ The enum name does not match the filename